sympy/algebras/quaternion.py

```diff
@@ -10,10 +10,50 @@
 from sympy.core.sympify import sympify, _sympify
 from sympy.core.expr import Expr
 from sympy.core.logic import fuzzy_not, fuzzy_or
+from sympy.core.numbers import pi
 
 from mpmath.libmp.libmpf import prec_to_dps
 
 
+def _elementary_axis(axis):
+    if axis == 'x':
+        return [1, 0, 0]
+    if axis == 'y':
+        return [0, 1, 0]
+    if axis == 'z':
+        return [0, 0, 1]
```

Comment:
```suggestion
    return [1 if i == axis for i in 'xyz']
```

Reply:
Nice one-liner! Wouldn't this be harder for others to read?

Reply:
The simple form seems pretty clear to me.

Reply:
> The simple form seems pretty clear to me.

I changed the code and put the suggested one-liner instead of calling the functions. This I can avoid adding two more function definitions to the code :)